lib/ansible/modules/windows/win_service.py

```diff
@@ -113,6 +113,20 @@
     - A newly created service will default to C(LocalSystem).
     type: str
     version_added: '2.3'
+  recovery_reset_interval:
+    description:
+      - Reset failure counter after this number of seconds.
+    type: int
+    default: 0
+    version_added: '2.10'
+  recovery_actions:
+    description:
+      - A list representing the recovery actions you want to set (3).
+      - First element of the list is the first failure actions/delay...
```

Comment:
I would simplify this by saying the list order is the order of actions to take, i.e. first entry is the first action and so on.